README.md

```diff
@@ -17,14 +18,16 @@ graph LR;
     C -->|Authorize| E(TDEI Database)
 ```
 
-- Auth service is responsible for Authenticating and Authorizing the HTTP requests. Also, verifies the access token,
-  api-key and generates the secret token
+- `Auth Service` is responsible for Authenticating and Authorizing the HTTP requests. Also, verifies the access token,
+  api-key on request and generates the secret token
   for intra micro-service communication.
-- Gateway service is first point of contact for clients. Every request to gateway is Authenticated & Authorized against
-  the Auth service.
-- For Authentication, Auth service will interact with keycloak service for verification.
-    - Successful authentication auth servive will return with access_token & refresh_token.
-- For Authorization, Auth service will interact directly with TDEI database.
+- `Gateway` service is first point of contact for clients. Every request to gateway is Authenticated & Authorized
+  against
+  the `Auth Service`.
+- For Authentication, `Auth Service` will interact with `Keycloak Service` for verification.

```

Comment:
For Authentication, `Auth Service` interacts with `Keycloak` for verification. Please write it in "present tense".